packages/neuron-ui/src/locales/en.json

```diff
@@ -991,6 +991,22 @@
         "view-concise-data": "View concise data",
         "view-raw-data": "View raw data"
       }
+    },
+    "dropdown": {
+      "placeholder": "Please select..."
+    },
+    "price-switch": {
+      "price": "Price",
+      "customPrice": "Custom Price",
+      "errorTip": "Price must not be less than {{minPrice}} shannons/byte, please re-enter",
+      "hintTip": "Suggested price is {{suggestFeeRate}} shannons/byte",
```

Comment:
`shannons/kB` is used widely, we'd better use it too

![image](https://user-images.githubusercontent.com/7271329/230344841-3a573e6e-9270-48f3-b891-948c54b986dd.png)

![image](https://user-images.githubusercontent.com/7271329/230344956-4f2589f1-77eb-4b19-94ee-d925a86fea60.png)